File: Four/PrivateVehicleLicense.php
<?php
include_once("License.php");

class PrivateVehicleLicense extends License {
    public function __construct() {
        parent::__construct("Private veicles", 2500000, 5);
    }

    public function calculateTotalCost(): float {
        return $this->getCost(); // The total cost for the private vehicle license is the base cost
    }
}
?>

File: java/Four/Four/PrivateVehicleLicense.java
public class PrivateVehicleLicense extends License {
    public PrivateVehicleLicense() {
        super("Private vehicles", 2500000.00, 5);
    }

    @Override
    public double calculateTotalCost() {
        return this.cost; // The total cost for the private vehicle license is the base cost
    }
}